action.yml

```diff
@@ -17,6 +17,12 @@ inputs:
         error: Fail the action with an error message
         ignore: Do not output any warnings or errors, the action does not fail
     default: 'warn'
+  retention-days:
+    description: >
+      Duration after which artifact will expire in days. 0 means using default retention.
```

Comment:
Would it be worth it to add `default: 0`?

Reply:
I don't think we need that because the server can handle no `retention-days`, and arguably we should leave it alone if we just want to take the default value to send less bytes over the network.  

I know it's a very small amount of bytes, I am hoping our usage volume becomes huge and any small saving counts.  ;)